TeslaCommand.go

```diff
@@ -1,179 +1,209 @@
-				if err != nil {
-					logger.Println(err)
-				}
-
-				// Send text
-				err = NotifyLib.SendText(logger, twilioSID, twilioToken, senderPhoneNumber, recipientPhoneNumber, body)
-				if err != nil {
-					logger.Println(err)
-				}
-			}
-		}
-		logger.Printf(waitmessage)
-	}
-}
+//
+// Brett Morrison, Februrary 2016
+// Lee Elson. June 2020 Modified to add Tesla wake up call with sleep time and to have a complete TO:, FROM:, SUBJECT:,MSG
+// in the message portion of the sendmail. This is necessary for some SMPT servers. Also commented out texting portion 
+// due to subscription cost. A second to: email address has been added allowing email-to-text if desired.
+// Also modified the loop so that the program ends if it finds the charge port door open. The program has
+// been changed to loop only if the door is closed and is designed to be run at a time when the vehicle **should** be home and attached.
+//
+// A program to alert if Tesla is not plugged in at a specified GeoFence
+//
+package main
+
+import (
+	"TeslaCommand/lib/HaversinFormula"
+	"TeslaCommand/lib/NotifyLib"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	tesla "github.com/jsgoecke/tesla"
+)
+
+// Magic clientid and clientsecret available here: http://pastebin.com/fX6ejAHd
+const clientid = "e4a9949fcfa04068f59abb5a658f2bac0a3428e4652315490b659d5ab3f35a9e"
+const clientsecret = "c75f14bbadc8bee3a7594412c31416f8300256d7668ea7e6e7f06727bfb9d220"
+
+var teslaLoginEmail string
+var teslaLoginPassword string
+var vehicleIndex int
+var geoFenceLatitude float64
+var geoFenceLongitude float64
+var mailServer string
+var mailServerPort int
+var mailServerLogin string
+var mailServerPassword string
+var fromAddress string
+var toAddress1 string
+var toAddress2 string
+var twilioSID string
+var twilioToken string
+var senderPhoneNumber string
+var recipientPhoneNumber string
+var radius int
+var checkInterval int
+var alertThreshold int
+
+func init() {
+	flag.StringVar(&teslaLoginEmail, "teslaLoginEmail", "", "Email for teslamotors.com account")
+	flag.StringVar(&teslaLoginPassword, "teslaLoginPassword", "", "Password for teslamotors.com account")
+	flag.IntVar(&vehicleIndex, "vehicleIndex", 0, "Index of vehicles in your account - If just 1 vehicle, use 0")
+	flag.Float64Var(&geoFenceLatitude, "geoFenceLatitude", 0.0, "Latitude of GeoFence Center")
+	flag.Float64Var(&geoFenceLongitude, "geoFenceLongitude", 0.0, "Longitude of GeoFence Center")
+	flag.StringVar(&mailServer, "mailServer", "", "SMTP Server hostname")
+	flag.IntVar(&mailServerPort, "mailServerPort", 25, "SMTP Server port number")
+	flag.StringVar(&mailServerLogin, "mailServerLogin", "", "SMTP Server login username")
+	flag.StringVar(&mailServerPassword, "mailServerPassword", "", "SMTP Server password")
+	flag.StringVar(&fromAddress, "fromAddress", "", "Alert send from email")
+	flag.StringVar(&toAddress1, "toAddress1", "", "Alert send to email1") //LSE mod to add second email address

```

Comment:
Remove all `// LSE` comments. They won't age well. The nature of Git / GitHub history tells the story.